.github/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+## Description of the change
+
+> Description here
+
+## Related Issue
+
+Issue #ID
```

Comment:
If you take a look on the right, you will see that Github allows linking a Github issue. I think this should be enough and we don't need to include it in the PR description as well.

WDYT?

Reply:
I've been working with GH for so long and never noted this field. Okay, we don't need to include this in the template.